### ISSUE TITLE
Error passing a positional arg after unpacked tuple when calling a vararg function

### ISSUE THREAD

**Issue by russelldavis** [microsoft/pyright#866]
```python
def foo1(a, b):
    print(a, b)

def foo2(*args):
    print(*args)

bar = (1,)
foo1(*bar, 2) # No problem
foo2(*bar, 2) # Pyright error: "Expected 1 positional argument"
```

See the error on the final line. (This is slightly different from #845, which I can confirm is fixed.)

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright & Plance.

**Comment by erictraut** [comment]
This is now implemented in Pyright 1.1.55, which I just published. It will also be included in the next release of Pylance.
